scipy/special/tests/test_support_alternative_backends.py

```diff
@@ -70,14 +80,16 @@ def test_support_alternative_backends(xp, data, f_name_n_args):
     dtype_np = getattr(np, dtype)
     dtype_xp = getattr(xp, dtype)
 
-    elements = dict(min_value=dtype_np(-10), max_value=dtype_np(10),
-                    allow_subnormal=False)
-    args_np = [np.asarray(data.draw(npst.arrays(dtype_np, shape, elements=elements)))
-               for shape in shapes]
+    # # To test the robustness of the alternative backend's implementation,
+    # # use Hypothesis to generate arguments
+    # elements = dict(allow_subnormal=False)  # consider min_value, max_value
+    # args_np = [np.asarray(data.draw(npst.arrays(dtype_np, shape, elements=elements)),
+    #                       dtype=dtype_np)
+    #            for shape in shapes]
```

Comment:
One could uncomment this code to find discrepancies between backends. This might not be very important as the various backends adopt the common special function library.